tests/slicetest.ark

```diff
@@ -0,0 +1,11 @@
+{
+    (import "StringMethods.ark")
+    # --------------------------
+    #           Slice
+    # --------------------------
+    (let slice-tests (fun () {
+        (assert (= "ABC" (slice "ABCDEFG" 3)) "Slice test 1 failed")
```

Comment:
those tests won't work since the new slice method is named `sliceStr` and it's taking 3 arguments, not 2